packages/react/src/components/header/Header.stories.tsx

```diff
@@ -337,56 +342,181 @@ export const MinimalWithLocalization = (args) => {
       <Header.SkipLink skipTo="#content" label="Skip To Content" />
+      <Header.UniversalBar primaryLinkText="Helsingin kaupunki" primaryLinkHref="#">
+        <Header.Link href="#" label="Uutiset" />
+        <Header.Link href="#" label="Asioi verkossa" />
+        <Header.Link href="#" label="Anna palautetta" />
+      </Header.UniversalBar>
+
       <Header.ActionBar
-        title={translations[lang]['header-title']}
-        titleAriaLabel={translations[lang]['header-aria-label']}
+        title="Helsingin kaupunki"
+        titleAriaLabel="Helsingin kaupunki"
         titleHref="https://hel.fi"
-        logo={<Logo src={logoSrcFromLanguageAndTheme(lang, args.theme)} alt={translations[lang]['header-title']} />}
+        logoAriaLabel="Service logo"
+        logoHref="https://hel.fi"
+        logo={<Logo src={logoSrcFromLanguageAndTheme('fi', 'custom')} alt="Helsingin kaupunki" />}
+        menuButtonAriaLabel="Menu"
       >
-        <Header.LanguageSelector languages={languages}>
-          <h3>{translations[lang]['header-menu-title']}</h3>
-          <Link external href="www.example.com">
+        <Header.LanguageSelector languages={languages} ariaLabel="Kielen valinta">
+          <h3>Tietoa muilla kielillä</h3>
+          <Link lang="fi" external href="www.example.com">
             Selkosuomi
           </Link>
-          <Link external href="www.example.com">
+          <Link lang="fi" external href="www.example.com">
             Viittomakieli
           </Link>
         </Header.LanguageSelector>
 
-        <Header.ActionBarItem
-          fullWidth
-          label={translations[lang]['header-search']}
-          icon={<IconSearch />}
-          id="action-bar-search"
-        >
-          <Header.Search
-            onChange={searchChangeAction}
-            onSubmit={searchSubmitAction}
-            label={translations[lang]['header-search']}
-          />
-        </Header.ActionBarItem>
-        <Header.ActionBarItem
-          label={translations[lang]['header-login']}
-          fixedRightPosition
-          icon={<IconUser />}
-          id="action-bar-login"
-        >
-          <h3>Kirjautumisvalinnat</h3>
+        <Header.ActionBarItem label="Haku" icon={<IconSearch />} id="action-bar-search">
+          <Header.Search label="Hae palvelusta" onChange={searchChangeAction} onSubmit={searchSubmitAction} />
         </Header.ActionBarItem>
       </Header.ActionBar>
 
       <Header.NavigationMenu>
-        <Header.Link href="#" label="Sosiaali- ja terveyspalvelut" />
-        <Header.Link href="#" label="Kasvatus ja koulutus" />
-        <Header.Link href="#" label="Asuminen" />
+        <Header.Link
+          href="#"
+          label="Sosiaali- ja terveyspalvelut"
+          onClick={(event) => event.preventDefault()}
+          active
+          dropdownLinks={[
+            <Header.Link
+              href="#"
+              label="Terveydenhoito"
+              active
+              dropdownLinks={[
```

Comment:
Any of the dropdowns don't have a themed background color.

Reply:
fixed